### ISSUE TITLE
Proposing django-autoslug

### ISSUE THREAD

**Issue by neithere** [jazzband/help#38]
Hi roadies,

I've been recently [advised](https://github.com/neithere/django-autoslug/pull/16#issuecomment-277950620) to move my semi-abandoned but still somewhat popular pet project [django-autoslug](http://django-autoslug.readthedocs.io/en/latest/) to JazzBand. There are a few pull requests which are necessary to merge, but I haven't had enough time for that for quite a while. I think it would be beneficial for the community to at least partially delegate maintenance to the organisation.

Questions:

1. Is it desirable for the organisation?
2. What steps do I need to take apart from creating CONTRIBUTING.md as per the Guidelines?
3. Is it possible to easily transfer all the issues and pending pull requests?

Thanks!

Regards,
Andy

**Comment by jezdez** [comment]
@neithere 

1. Yeah, this is desirable assuming you follow the [guidelines](https://jazzband.co/about/guidelines). django-autoslug seem to be well-suited for that.

2. Once you transferred the repo to the Jazzband organization via [GitHub's transfer feature](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/#transferring-a-repository-to-another-user-account-or-to-an-organization). The Jazzband bot will automatically open a ticket here in the roadies repo that with a TODO list that you can work through

3. Yes, all issues and pull requests will be transferred as well if you use GitHub's repo transfer feature. Links to those issues and PRs will also keep working and redirect to their new URL.

**Comment by justinmayer** [comment]
Hi Andy. I agree with @jezdez that `django-autoslug` seems well-suited for Jazzband. Would you please take a moment and transfer the repo to the Jazzband organization via [GitHub's transfer feature](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/#transferring-a-repository-to-another-user-account-or-to-an-organization)? That way we can collectively work toward getting a new `django-autoslug` version released to PyPI that works well with current Django releases. 😊

**Comment by justinmayer** [comment]
@neithere: Please feel free to add me as an owner of the current `django-autoslug` repo so I can assist with transferring it here to JazzBand. Would that be helpful?

**Comment by tony** [comment]
@neithere Did you give @justinmayer ownership yet?

**Comment by jezdez** [comment]
@justinmayer Thank you for the offer to help. Anything that you need, please let me know.

PS: I just opened #75 to document the process of repo transfer better.

**Comment by tony** [comment]
Careful on this one, django-autoslug is LGPL. This adds additional complications for Python code / scripting language and can result in unforeseen issues. Basically, LGPL has a lot of additional language that's ambiguous and *could* result in an accusation of creating a derivative.

Generally speaking, python libraries, and for that matter, Django libraries, stick to MIT/BSD/ISC/etc. 

It is strongly advised to use a permissive license.

**Comment by neithere** [comment]
Sorry for such a long pause. Frankly, I don't remember why did I not finalize the transfer last time. I'd like to do it now, as I have a little bit of spare time.

1. LGPL [should not be a problem](https://stackoverflow.com/questions/8580223/using-python-module-on-lgpl-license-in-commercial-product), and it was definitely a conscious choice. _Free_ software is better than just open source.
2. I understand that many potential users are scared of [L]GPL for all the good and bad reasons. In the enterprise world it may be harder to get a legal approval for such a library, and it merely means a shift in focus to a similar one, so Free Software doesn't really win in that case.
3. Anyway, it does make sense to use the same license as Django itself.

Conclusion: I would indeed change the license to BSD. However, this would [require obtaining permission from all contributors](https://stackoverflow.com/questions/1409666/switching-license-from-gpl-to-bsd-in-later-time). I don't have the time for that, so it can be done after the transfer.

**Comment by neithere** [comment]
@justinmayer is it OK that I've recently transferred that repo to you?

**Comment by justinmayer** [comment]
@neithere: Yes, thank you. I accepted the transfer. I'm traveling for the next few weeks but will do my best to turn my attention to this endeavor as soon as my schedule has settled down. Thanks again!

**Comment by W7PEA** [comment]
I was wondering if there is a schedule for the transfer, planning to do some 2.0 upgrades that will need this library and if there is anything the community can help with?

thanks!

**Comment by camilonova** [comment]
@justinmayer how can we do this?

**Comment by gregsadetsky** [comment]
Small ping to see if this is still being discussed/considered? Thank you!

**Comment by camilonova** [comment]
@justinmayer ping
